src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2623,6 +2627,19 @@ public bool ExtensionsCollapsed
             }
         }
 
+        /// <summary>
+        /// A feature flag controlling the appearance of the Dynamo home navigation page
+        /// </summary>
+        public bool UseNewDynamoHomePage
+        {
+            get
+            {
+                return true;
+                // TODO: replace with a similar flag to the below when deploying the feature
+                //return DynamoModel.FeatureFlags?.CheckFeatureFlag("NodeAutocompleteMachineLearningIsBeta", false) ?? false;
```

Comment:
Can you use feature flag `IsNewAppHomeEnabled`?

Reply:
Done, working great!